library/oneview_hypervisor_cluster_profile.py

```diff
@@ -0,0 +1,176 @@
+###
+# Copyright (2016-2020) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_hypervisor_cluster_profile
+short_description: Manage OneView Hypervisor Cluster Profiles resources.
+description:
+    - Provides an interface to manage Hypervisor Cluster Profiles resources. Can create, update, and delete.
+version_added: "2.4"
+requirements:
+    - "python >= 3.4.2"
+    - "hpOneView >= 5.1.0"
+author: "Venkatesh Ravula (@VenkateshRavula)"
+options:
+    state:
+        description:
+            - Indicates the desired state for the Hypervisor Cluster Profiles resource.
+              C(present) will ensure data properties are compliant with OneView.
+              C(absent) will remove the resource from OneView, if it exists.
+        choices: ['present', 'absent']
+    data:
+        description:
+            - List with the Hypervisor Cluster Profiles properties.
+        required: true
+
+extends_documentation_fragment:
+    - oneview
+    - oneview.validateetag
+'''
+
+EXAMPLES = '''
+- name: Create a Hypervisor Cluster Profile
+  oneview_hypervisor_cluster_profile:
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 1200
+    state: present
+    data:
+      name: 'hcp'
+      path: 'DC1'
+      hypervisorType: 'Vmware'
+      hypervisorManagerUri: '/rest/hypervisor-manager/rdy-dfdf12'
+      hypervisorHostProfileTemplate:
+        serverProfileTemplateUri: '/rest/server-profile-template/2323-32323'
+        hostprefix: 'test-host'
+
+- name: Update the Hypervisor Cluster Profile name to 'hcp Renamed'
+  oneview_hypervisor_cluster_profile:
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 1200
+    state: present
+    data:
+      name: 'hcp Renamed'
+      path: 'DC1'
+      hypervisorType: 'Vmware'
+      hypervisorManagerUri: '/rest/hypervisor-manager/rdy-dfdf12'
+    params:
+      force: True
+
+- name: Ensure that the Hypervisor Cluster Profile is absent
+  oneview_hypervisor_cluster_profile:
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 1200
+    state: absent
+    data:
+      name: 'hcp'
+
+'''
+
+RETURN = '''
+hypervisor_manager:
```

Comment:
It should be Hyper cluster profile, not hypervisor manager.

Reply:
done